app/controllers/homes_controller.rb

```diff
@@ -1,4 +1,11 @@
 class HomesController < ApplicationController
   def show
   end
+
+  private
+
+  def redirections
+    @_redirections ||= Redirection.in_ring_order
+  end
+  helper_method :redirections
```

Comment:
Sure! I did it this way because Sean Griffin tries to avoid ivars in controller methods. I can't remember the exact reasoning but it made sense in my head when I heard it.